lib/internal/modules/esm/loader.js

```diff
@@ -109,6 +110,28 @@ let hooksProxy;
  * @typedef {ArrayBuffer|TypedArray|string} ModuleSource
  */
 
+/**
+ * @typedef {Object} LoadResult
```

Comment:
This exists within `customization_hooks.js`: `ModuleLoadResult`

related: https://github.com/nodejs/node/pull/56454